docs/docs/ProgrammingGuide/run-on-k8s.md

```diff
@@ -1,18 +1,18 @@
-The [Apache Spark on Kubernetes](https://apache-spark-on-k8s.github.io/userdocs/index.html) project enables
+The [Apache Spark on Kubernetes](https://spark.apache.org/docs/2.3.0/running-on-kubernetes.html) project enables
 native support for submitting Spark application to a kubernetes cluster. As a deep learning library for Apache
 Spark, BigDL can also run on Kubernetes by leveraging Spark on Kubernetes.
 
 ---
 ## **Prerequisites**
 
-1. You need to have a running Kubernetes cluster that support Spark on Kubernetes. See [here](https://apache-spark-on-k8s.github.io/userdocs/running-on-kubernetes.html#prerequisites)
-
-2. You need to spin up the **resource staging server** for dependency management. See [Dependency Management](https://apache-spark-on-k8s.github.io/userdocs/running-on-kubernetes.html#dependency-management) (This is optional if all your application dependencies are
-packaged into your own custom docker image or resides in remote locations like HDFS. See [Dependency Management Without The Resource Staging Server](https://apache-spark-on-k8s.github.io/userdocs/running-on-kubernetes.html#dependency-management-without-the-resource-staging-server))
+You need to have a running Kubernetes cluster that support Spark on Kubernetes. See [here](https://spark.apache.org/docs/2.3.0/running-on-kubernetes.html#prerequisites)
 
 ---
 ## **Docker images**
+For Spark2.3,
+BigDL ships with a Dockerfile that can be found in the ```kubernetes/dockerfiles/``` directory.
 
+For Spark2.2,
```

Comment:
Remove Spark 2.2? As the k8s is officially supported in Spark 2.3.

Reply:
Ok.